### PR TITLE
com.github.philip-scott.notes-up 2.0.3

### PR DESCRIPTION
**Pull request by danrabbit**
Repository: [Philip-Scott/Notes-up](https://github.com/Philip-Scott/Notes-up)
Submitted By: @Philip-Scott

## Review Checklist

- [x] App opens
- [x] https://github.com/Philip-Scott/Notes-up/issues/388
- [x] Does what it says
- [x] Categories match

### AppData
- [x] Name is unique and non-confusing
- [x] Matches description
- [x] Matches screenshot
- [x] https://github.com/Philip-Scott/Notes-up/issues/391
- [x] https://github.com/Philip-Scott/Notes-up/issues/387
- [x] Release tag with matching version and YYYY-MM-DD date
- [x] Custom colors meet WCAG A contrast or greater
- [x] OARS info matches

### Flatpak
- [x] Uses elementary runtime
- [x] Sandbox permissions are reasonable

### CODE REVIEW

File: applications/com.github.philip-scott.notes-up.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/Philip-Scott/Notes-up.git",
+  "commit": "442bd28ced08332c6f9574c948993fac232a5465",
+  "version": "2.0.3"
```
Comment (Philip-Scott) [inline applications/com.github.philip-scott.notes-up.json:4]:
```suggestion
  "version": "2.0.4"
```

File: applications/com.github.philip-scott.notes-up.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/Philip-Scott/Notes-up.git",
+  "commit": "442bd28ced08332c6f9574c948993fac232a5465",
```
Comment (Philip-Scott) [inline applications/com.github.philip-scott.notes-up.json:3]:
```suggestion
  "commit": "29e72f17232a0c5c0671ce58c2b5832ca5041879",
```

**Review by Philip-Scott — CHANGES_REQUESTED** [review]
I don't have push access to this repo, but I'm leaving the new release info as a suggestion for a hopefully easy change :)

**Comment by Philip-Scott** [comment]
Should this file be renamed to `com.github.philip_scott.notes-up.json` after the next Notes-Up release? Flatpak does not allow for a `-` before the last section of the RDNN

**Comment by danrabbit** [comment]
@Philip-Scott yeah your ID will need to be consistent throughout, so I'd suggest switching everything over to use the underscore

File: applications/com.github.philip_scott.notes-up.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/Philip-Scott/Notes-up.git",
+  "commit": "29e72f17232a0c5c0671ce58c2b5832ca5041879",
+  "version": "2.0.4"
```
Comment (Philip-Scott) [inline applications/com.github.philip_scott.notes-up.json:4]:
```suggestion
  "version": "2.0.5"
```

Comment (Philip-Scott) [inline applications/com.github.philip_scott.notes-up.json:4]:
```suggestion
  "version": "2.0.5"
```

File: applications/com.github.philip_scott.notes-up.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/Philip-Scott/Notes-up.git",
+  "commit": "29e72f17232a0c5c0671ce58c2b5832ca5041879",
```
Comment (Philip-Scott) [inline applications/com.github.philip_scott.notes-up.json:3]:
```suggestion
  "commit": "47d311735e1b59e11e20e02b12a7b9aa01ccf89c",
```

**Review by Philip-Scott — COMMENTED** [review]
New release should have the new RDNN

**Comment by danrabbit** [comment]
@Philip-Scott everything works now, but you have non-default window controls :upside_down_face:

File: applications/com.github.philip_scott.notes-up.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/Philip-Scott/Notes-up.git",
+  "commit": "47d311735e1b59e11e20e02b12a7b9aa01ccf89c",
```
Comment (Philip-Scott) [inline applications/com.github.philip_scott.notes-up.json:3]:
```suggestion
  "commit": "7c7e26a8e67a49a92885f304fc5fa8ccb4be1d05",
```

File: applications/com.github.philip_scott.notes-up.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+  "version": "2.0.5"
```
Comment (Philip-Scott) [inline applications/com.github.philip_scott.notes-up.json:4]:
```suggestion
  "version": "2.0.6"
```

**Review by Philip-Scott — COMMENTED** [review]
Fixed screenshots on the latest release :)

**Comment by danrabbit** [comment]
Let's goooo :rocket:

**Pull request merged**